pybamm/models/submodels/tortuosity/bruggeman_tortuosity.py

```diff
@@ -8,30 +8,40 @@
 
 class Bruggeman(BaseModel):
     """Submodel for Bruggeman tortuosity
+    phase : str
+        The phase in which the model is implemented ("Electrolyte" or "Electrode")
+    options : dict, optional
+        A dictionary of options to be passed to the model.
 
     **Extends:** :class:`pybamm.tortuosity.BaseModel`
     """
```

Comment:
```suggestion
    """Submodel for Bruggeman tortuosity

    Parameters
    ----------
    param : parameter class
        The parameters to use for this submodel
    phase : str
        The material for the model ('electrolyte' or 'electrode').
    options : dict, optional
        A dictionary of options to be passed to the model.

    **Extends:** :class:`pybamm.tortuosity.BaseModel`
    """
```

I don't know exactly why, but this seems to fix the docs issue